README.md

```diff
@@ -6,14 +6,29 @@ Run [Azure IoT Sample](demo/sample_azure_iot/sample_azure_iot.c))
 
 ### Prerequisites
 
-- CMake (> 3.13)
-- Ninja
+- [CMake](https://cmake.org/download/) (> 3.13)
+- [Ninja build system](https://github.com/ninja-build/ninja/releases)
+- [GNU Arm Embedded Toolchain](https://developer.arm.com/tools-and-software/open-source-software/developer-tools/gnu-toolchain/gnu-rm/downloads) (>v9)
 
 ### Quickstart
 
-1. git clone https://github.com/Azure-Samples/iot-middleware-freertos-samples.git
-2. cd iot-middleware-freertos-samples
-3. git checkout develop_higupt_sample
+#### Install prerequisites on Linux (or Windows Subsystem for Linux)
+
+`./.github/scripts/install_software.sh`
+
+#### Install prerequisites on Windows
+Ensure that cmake, ninja and the ARM toolset binaries are available in the `PATH` environment variable.
+
+You may also need to enable long path support for both Windows and git:
+- Winodws: https://docs.microsoft.com/windows/win32/fileio/maximum-file-path-limitation?tabs=cmd#enable-long-paths-in-windows-10-version-1607-and-later
```

Comment:
`Winodws` -> `Windows`

Reply:
Will make the change together with how to run the demos.